config/const.go

```diff
@@ -1,10 +1,23 @@
 package config
 
 const (
-	PathGetDeploymentJTI = "/deploy/nextJTI/:serviceName"
-	PathAddDeployment    = "/deploy"
+	PathGetDeploymentJTI        = "/deploy/nextJTI/:serviceName"
+	PathAddDeployment           = "/deploy"
+	PathGetLatestDeploymentInfo = "/deployment/:serviceName"
 )
 
 const (
 	ErrorServiceNotFound string = "SERVICE_NOT_FOUND"
 )
+
+const (
+	envComposeFile           = "COMPOSE_FILE"
+	envComposeProjectName    = "COMPOSE_PROJECT_NAME"
+	envHostComposeWorkingDir = "HOST_COMPOSE_WORKING_DIR"
```

Comment:
Why not change this LOC? https://github.com/CleverseAcademy/cd-compose-deployment/pull/4/files#diff-8e2571d8da443c5699ef0c75ddb3a94e76f2582b5973bb81f9746b636293db15R38